src/hooks/useGovernanceReview.ts

```
@@ -0,0 +1,69 @@
+import { useMutation } from '@tanstack/react-query';
+import { useAccount } from './useAccount';
+import { useCallback } from 'react';
+import { governanceReview } from '@/lib/api';
+import { GovernanceReviewFormValues } from '@/components/governance-review/GovernanceReviewForm';
+
+interface UseGovernanceReviewProps {
+  onSignaturePending?: () => void;
+  onReviewPending?: () => void;
+  onSuccess?: () => void;
+  onError?: (error: Error) => void;
+}
+
+export const useGovernanceReview = ({
+  onSignaturePending,
+  onReviewPending,
+  onSuccess,
+  onError,
+}: UseGovernanceReviewProps) => {
+  const { signStateMessage, account } = useAccount();
+
+  const handleGovernanceReview = useCallback(
+    async (id: string, data: GovernanceReviewFormValues) => {
+      try {
+        onSignaturePending?.();
+        const signature = await signStateMessage(`Governance Review ${id} ${data.allocatorType}`);
+
+        const pubKey =
+          account?.wallet?.getPubKey() || Buffer.from('0x0000000000000000000000000000000000000000');
```

Is this the best default? If there's no wallet that's a clear error we can signal in, whereas an invalid default might mask errors later?